CHANGES.rst

```diff
@@ -1,11 +1,8 @@
 ChangeLog
 =========
 
-Please note:
-the change log will only get updated after first release - for now please use the
-`commit log <https://github.com/certbot/certbot/commits/master>`_.
+To see the changes in a given release, view the issues closed in a given
+release's GitHub milestone:
 
-To see the changes in a given release, inspect the github milestone for the
-release.  For instance:
-
-https://github.com/certbot/certbot/issues?utf8=%E2%9C%93&q=milestone%3A0.3.0
+ - [Past releases](https://github.com/certbot/certbot/milestones?state=closed)
```

Comment:
Because this is an reStructuredText file rather than markdown, these links aren't displaying properly. In case you're not familiar with the syntax, here's [how you create links in rst files](http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets).